kqueen/engines/jenkins.py

```diff
@@ -74,39 +106,40 @@ def _get_provision_job_builds(self):
         """
         return self.client.get_job_info(self.provision_job_name, depth=1)
 
-    @classmethod
-    def engine_status(cls, **kwargs):
+    def _parameter_exist(self, parameter):
         """
-        Implementation of :func:`~kqueen.engines.base.BaseEngine.engine_status`
-        """
-        conn_kw = {
-            'username': kwargs.get('username', config.get('JENKINS_USERNAME')),
-            'password': kwargs.get('password', config.get('JENKINS_PASSWORD')),
-            'timeout': 10
-        }
-        status = config.get('PROVISIONER_UNKNOWN_STATE')
-        try:
-            client = jenkins.Jenkins(config.get('JENKINS_API_URL'), **conn_kw)
-            auth_verify = client.get_whoami()
-            if auth_verify:
-                status = config.get('PROVISIONER_OK_STATE')
-        except Exception as e:
-            logger.exception('Could not contact JenkinsEngine backend: ')
-            status = config.get('PROVISIONER_ERROR_STATE')
-        return status
+        Check, that defined key parameter exist in Job
```

Comment:
that defined key parameter exist in Job -> that key parameter, defined by the user exists in the specified Job